lib/elixir/lib/string.ex

```diff
@@ -1121,9 +1121,28 @@ defmodule String do
       iex> String.replace("a,b,c", ",", "[]", insert_replaced: [1, 1])
       "a[,,]b[,,]c"
 
+  When an empty string is provided as a `pattern`, the function will treat it as
+  an implicit empty string between each character and the string will be
```

Comment:
Should we say `codepoint` instead of `character` here?

Reply:
IMO character is alright, but if we want to be more specific it should be `grapheme`, not `codepoint`.

Reply:
As I see we use character in other places.

Reply:
I guess we need to go over `String` docs since our use of "character" is inconsistent: usually it stands for grapheme, some times—codepoint. 🤔

Reply:
We should use grapheme.